le-parole-della-pubblica-amministrazione/w.rst

```diff
@@ -2,5 +2,5 @@ W
 =
 
 wi-fi
```

Comment:
Il nome corretto dovrebbe essere "Wi-Fi", trattandosi di uno standard (come da [pagina della Wi-Fi Alliance](https://www.wi-fi.org/)). Propongo di indicare "Wi-Fi", spiegando che "wi-fi" tutto in minuscolo può comunque essere accettabile.